<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_Close">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>460</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox { 
     border: 1px solid gray; 
     border-radius: 1px; 
 } </string>
    </property>
    <property name="title">
     <string>Serial Port Settings</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QComboBox" name="comboBox_COMPort">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Select Port</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM5</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_BaudRate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>122</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Select Baud Rate</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>460800</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton_Connect">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#550000;&quot;&gt;Click to connect to selected COM port with selected baudrate.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_StatusSerialConnect">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>false</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 0, 0)</string>
    </property>
    <property name="text">
     <string>Serial Port Not Connected.</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>480</y>
      <width>661</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_WindowAliveIndicator">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>505</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(85, 0, 0)</string>
    </property>
    <property name="text">
     <string>Window Alive</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_StatusRxCount">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>510</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(85, 0, 0)</string>
    </property>
    <property name="text">
     <string>Rx Count:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>221</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox { 
     border: 1px solid gray; 
     border-radius: 1px; 
 } </string>
    </property>
    <property name="title">
     <string>Robot Positions</string>
    </property>
    <widget class="QLabel" name="label_4a">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>100</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>label_4a</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3a">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>100</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>label_3a</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_1a">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>100</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>label_1a</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4b">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>label_4b</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2a">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>100</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>label_2a</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_1b">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>label_1b</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2b">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>label_2b</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3b">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>label_3b</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4a1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>label_4a1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4b1">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>label_4b1</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox { 
     border: 1px solid gray; 
     border-radius: 1px; 
 } </string>
    </property>
    <property name="title">
     <string>Guidewire Attributes</string>
    </property>
    <widget class="QLabel" name="label_5a">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>100</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>label_5a</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6b">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>label_6b</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6a">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>100</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>label_6a</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5b">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>label_5b</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5c">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>label_5c</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox { 
     border: 1px solid gray; 
     border-radius: 1px; 
 } </string>
    </property>
    <property name="title">
     <string>Force/Torque Field Attributes</string>
    </property>
    <widget class="QLabel" name="label_7a">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>100</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>label_7a</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7b">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>label_7b</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7a1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>label_7a1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7b1">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>label_7b1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7b3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>label_7b3</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7b2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>label_7b2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7a2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>label_7a2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7a3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>label_7a3</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox { 
     border: 1px solid gray; 
     border-radius: 1px; 
 } </string>
    </property>
    <property name="title">
     <string>Angle Difference</string>
    </property>
    <widget class="QLabel" name="label_8a">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>100</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>label_8a</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8b">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>label_8b</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9b">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>label_9b</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9a">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>100</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>label_9a</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox { 
     border: 1px solid gray; 
     border-radius: 1px; 
 } </string>
    </property>
    <property name="title">
     <string>Hatic Feedback</string>
    </property>
    <widget class="QLabel" name="label_10a">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>label_10a</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10b">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>label_10b</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11a">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>label_11a</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11b">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>label_11b</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_ImgProcRate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>false</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 0, 0)</string>
    </property>
    <property name="text">
     <string>Image Processing Rate = </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_HapRendRate">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>490</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>false</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 0, 0)</string>
    </property>
    <property name="text">
     <string>Haptic Rendering Rate = </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_SerComRate">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>490</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>false</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 0, 0)</string>
    </property>
    <property name="text">
     <string>Serial Communication Rate =</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>669</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Developers"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout_Developers">
   <property name="text">
    <string>About Developers</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
